datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1887,11 +1887,13 @@ pub struct Filter {
     pub predicate: Expr,
     /// The incoming logical plan
     pub input: Arc<LogicalPlan>,
+    // Selectivity
```

Comment:
I think it's beter to move this to the physical plan `FilterExec` only, so we don't have to pass it in logica plan.